src/zmalloc.c

```diff
@@ -90,6 +90,7 @@ void zlibc_free(void *ptr) {
 
 #define thread_local _Thread_local
 
+#define PADDING_ELEMENT_NUM CACHE_LINE_SIZE / sizeof(size_t) - 1
```

Comment:
Nit: Parentheses is common practice for macro.

```suggestion
#define PADDING_ELEMENT_NUM (CACHE_LINE_SIZE / sizeof(size_t) - 1)
```